packages/dd-trace/src/proxy.js

```diff
@@ -44,6 +44,11 @@ class Tracer extends BaseTracer {
             analyticsSampler.enable()
           }
 
+          // dirty require for now so zero appsec code is executed unless explicitely enabled
+          if (config.appsec && config.appsec.enabled) {
```

Comment:
`config.appsec` will always be defined and doesn't need to be checked.

Reply:
Not in the tests ! would you rather have the appsec object be hardcoded in the tests ?